x-pack/plugins/fleet/server/services/epm/packages/get.ts

```diff
@@ -415,6 +421,10 @@ export async function getPackageInfo({
   ignoreUnverified?: boolean;
   prerelease?: boolean;
 }): Promise<PackageInfo> {
+  const cacheResult = getPackageInfoCache(pkgName, pkgVersion);
```

Comment:
Curious to get your though on that pattern for caching, it seems easier to use that passing packageInfo through all codebase